tutorials/fundamentals/filter-key.md

```diff
@@ -0,0 +1,36 @@
+To filter an object using the keys, you can use JavaScript's `Object.hasOwnProperty()` method.
+
+```javascript
+const obj = { name: 'Masteringjs', location: 'Florida', help: true };
+let newObj = {}
+if(obj.hasOwnProperty('name')) {
```

Comment:
There's no loop here, so what is this filtering?

Reply:
It's making a new object with property name from the original object that had name, location, and help